apps/aecore/src/aec_conductor.erl

```diff
@@ -947,7 +947,8 @@ setup_loop(State = #state{ consensus = Cons }, RestartMining, IsLeader, Origin)
     State1 = State#state{ consensus = Cons#consensus{ leader = IsLeader } },
     State2 =
         case Origin of
-            block_created when IsLeader ->
+            Origin when IsLeader, Origin =:= block_created
+                           orelse Origin =:= block_received ->
```

Comment:
Strictly speaking, out of scope of this PR though still re semantics of this block received from HTTP internal API:
Is this block published to network peers (`block_to_publish`)? Is it meant to? In any case, on the assumption that this key block is a key block that starts a generation of microblocks signed by this node, how are peers meant to get this keyblock when a successor microblock is published (`micro_block_created`)? Do peers get it?